peers/app_request_network.go

```diff
@@ -145,8 +156,10 @@ func (n *AppRequestNetwork) ConnectPeers(nodeIDs set.Set[ids.NodeID]) set.Set[id
 	// through connectedPeers for already tracked peers, just iterate through the full list,
 	// re-adding connections to already tracked peers.
 
+	startInfoAPICall := time.Now()
 	// Get the list of peers
 	peers, err := n.infoAPI.Peers(context.Background())
+	n.metrics.setInfoAPICallLatencyMS(float64(time.Since(startInfoAPICall).Milliseconds()))
```

Comment:
Could we create a receiver function `func (n *AppRequestNetwork) setInfoAPICallLatencyMS(t time.Time)` to extract all this conversion?